app/components/concerns/booking_expiry.rb

```diff
@@ -0,0 +1,9 @@
+module BookingExpiry
```

Comment:
I am not sure where the best place for concerns such as these. This type of helper is used in guards, policies, components and services. Perhaps app/models/concerns is the best place.

Reply:
Try `app/models/concerns` for now.